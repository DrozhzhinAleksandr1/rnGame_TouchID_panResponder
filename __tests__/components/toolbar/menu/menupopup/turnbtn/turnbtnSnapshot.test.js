import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import {TurnBtn}  from '../../../../../../app/components/toolbar/menu/menupopup/turnBtn/turnbtn';
import { localization } from '../../../../../../app/common/localization';

it('create snapshot MenuPopUp ', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<TurnBtn CurrentLanguage={localization.ru}/>);
    expect(toolbarSnapshot).toMatchSnapshot();
});
