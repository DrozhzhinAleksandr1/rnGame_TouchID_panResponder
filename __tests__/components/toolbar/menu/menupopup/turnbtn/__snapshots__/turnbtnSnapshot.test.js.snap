// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot MenuPopUp  1`] = `
<TouchableWithoutFeedback
  accessibilityLabel="BtnTurnAll"
  onPress={[Function]}
  testID="BtnTurnAll"
>
  <Component
    style={
      Object {
        "flexDirection": "row",
        "height": 24,
        "marginLeft": 16,
        "marginTop": 40,
      }
    }
  >
    <Icon_arrow
      style={
        Object {
          "height": 24,
          "width": 24,
        }
      }
    />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Roboto-Light",
          "fontSize": 18,
          "lineHeight": 20,
          "marginLeft": 16,
          "textAlignVertical": "bottom",
        }
      }
    >
      Свернуть все
    </Text>
  </Component>
</TouchableWithoutFeedback>
`;
